### DISCUSSION TITLE
Feedback version 20210923

### DISCUSSION THREAD

**Discussion by edzob** [JRBliekendaal/master-thesis#46] [General]
Feedback in discussions .. so that points that are accepted and have shared understanding can be promoted to issues.
1. Introduction:  Reads great. Great start
2. Introduction: Tip.. active voice in the I vorm and not in the reseachers.. "In this thesis, I define Enterprise Architecture..." etc
3. Introduction: "Defines the EA sucess factors for contribution to become antifragile" .. as an organisation as a whole? the of what should be added
4. The structure of the thesis: The structure of this thesis
5. The structure of the thesis: Chater 3 explains the methodology --> the research methodology
6. Introduction of the public sector: the term "voluntary sector" is not that common. perhaps that nonproffit sector is more fitting.
7. Introduction of the public sector: help the reader if the stated analysis is generic (global) or specific for the netherlands
8. Introduction of the concept EA: After the module EA it would add value to use references that have more standing.. A bit more oemphf adds to the readers perception
9. Problem statement introduces concepts as Application and Information architecture to reason with. When you start a logic reasoning it is a good practice to define the term first. So that the reader knows it has the same mental model as you.
10. Research Relefance: great piece to read. Looking forward to the next iterations. small thing: Public Sector is now in small letters, and camel cased in the previous sections.
11. Open en Closed: The definition provided is focused on the emergent behaviour. CAS might be more. The research group of the VUB might provide some nice definitions. 
12. Open en Closed: Nice one on second law of thermodynamics. good to discuss virtual/face-to-face how to proceed with this line of reasoning.
13. Complexity theory: good to introduce your definition of "complex" since this helps
14. Antifragile: When using Kastner it is good to point out why not use EAAL. They will ask. Since EAAL is a superset off antifragile attributes. 
15. Do know that the focus of O'reilly was/is on software engineering. And then adds business to the flavour. for example: "translate the impact on the software decomposition" .. does this match your view on organisations and EA as selected in this research?
16. Resilient system are by definiition able to survive disruption can be formulated more concise. 
17. Three schools of lapalme: I think he published also a revised, renewed paper on this with the same three schools.. I need to check that.
18. rest looks promossion. Love the detail of the penns and the notebook :)

**Comment by JRBliekendaal** [comment]
Placed some already on the issue list and planned them on the board. 

I am not forgetting eaal the current background chapter is only a big collection of brain farts and citations. It gives me structure. Eaal will be in there I know it is a meta model even better I need it because of the learning part. 

About scoping what of the public sector. I am going for local governments but I am going to define the public sector as a system of systems. Of it contains more than only public sector. This is the main reason why I cited that part on the different collaboration models. At the last iBestuur congres they where talking a lot on working together in joined ventures. So when there is a joined venture of a public and a private enterprise that is delivering a service to a citizen than I have the opinion you will have a system of systems consisting out of a private sector system and a public sector system. Even better the new constellation is that public sector or private ; I think that when the delivery is in the public sector then de private enterprise is for the part that works together also part of the private sector 😎 this is exactly the scope that I want to have. ISV+agency (hybrid) is part of public sector.

**Reply by edzob** [reply]
Nice! on towards the iterations! :)

**Comment by edzob** [comment]
Snowballing
In methodologie (version 20.00 20210923) snowball was mentioned very shortly.
doing the complete process creates a large corpus and it officially also includes a citation-matrix.
I did not do this, i think this is very much work but a small matrix adds value.. 
something to think about and customize for the route taken.

**Reply by JRBliekendaal** [reply]
I will look into it. before we know it we have a research approach of a Ph.D. 🗡️  But I do not care.. I can spend a lot of time in the next half year.

**Reply by edzob** [reply]
when you explicit mention what you did do and what you did not do all is good :)

**Reply by JRBliekendaal** [reply]
> when you explicit mention what you did do and what you did not do all is good :)

Exactly. That was what I was thinking about. My main research is not literature research. But the matrix also should not be that difficult to produce 😬

**Comment by JRBliekendaal** [comment]
1. Nothing needed
2. Done
3. wip
4. Done
5.  Done
6. Done
7. wip
8. Nothing needed at this moment
9. wip
10. All in small caps (replaced it)
11. chapter is still scattered and work in progress. Nothing needed at this moment
12. chapter is still scattered and work in progress. Nothing needed at this moment
13. chapter is still scattered and work in progress. Nothing needed at this moment
14. chapter is still scattered and work in progress. Nothing needed at this moment. Not going to use Kastner as central point. But kastner has some good articulations that are nice to cite and even better it makes the story of (Botjes, 2020) only stronger. Only one source is not a good idea ;) EAAL btw is one of the inputs for the success factors. So do not worry ;) I started with copy past from literature after I did read your thesis ;)
15. chapter is still scattered and work in progress. Nothing needed at this moment. thx for the info
16. chapter is still scattered and work in progress. Nothing needed at this moment thx for the info
17. chapter is still scattered and work in progress. Nothing needed at this moment thx for the info
18. chapter is still scattered and work in progress. Nothing needed at this moment thx for the info
